ios/browser/api/url_sanitizer/url_sanitizer_service.mm

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/ios/browser/api/url_sanitizer/url_sanitizer_service+private.h"
+
+#include <string>
+
+#include "base/memory/raw_ptr.h"
+#include "brave/components/url_sanitizer/browser/url_sanitizer_service.h"
+#import "net/base/mac/url_conversions.h"
+#include "url/gurl.h"
+
+@interface URLSanitizerService () {
+  raw_ptr<brave::URLSanitizerService> urlSanitizer_;  // NOT OWNED
+}
+
+@end
+
+@implementation URLSanitizerService
+
+- (instancetype)initWithURLSanitizerService:
+    (raw_ptr<brave::URLSanitizerService>)urlSanitizer {
+  self = [super init];
+  if (self) {
+    urlSanitizer_ = urlSanitizer;
+  }
+  return self;
+}
+
+- (nullable NSURL*)sanitizedURL:(NSURL*)url {
```

Comment:
`sanitizedURL` -> `sanitizeURL`

Reply:
there isn't necessarily a chromium convention here, what I wanted was consistency in naming